packages/@jsii/java-runtime/project/src/main/java/software/amazon/jsii/JsException.java

```diff
@@ -0,0 +1,33 @@
+package software.amazon.jsii;
+
+/*
+ * Represents an irrecoverable error, such as a
```

Comment:
I think this should be "potentially recoverable" right as JSError's may just be basic input validation etc on the lib side.